File: backend/src/main/java/com/soufiane/open_data/dtos/dataProviderOrganisationMember/DataProviderOrganisationMemberRequest.java
package com.soufiane.open_data.dtos.dataProviderOrganisationMember;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import java.time.LocalDateTime;

@Getter @Setter @NoArgsConstructor @AllArgsConstructor
public class DataProviderOrganisationMemberRequest {
    private String firstName;
    private String lastName;
    private String email;
    private String createdBy;
    private String updatedBy;
    private LocalDateTime createdOn;
    private LocalDateTime updatedOn;
}

File: frontend/src/app/models/DataProviderOrganisationMemberRequest.ts
import { DataProviderOrganisationRequest } from "./DataProviderOrganisationRequest";

export interface DataProviderOrganisationMemberRequest {
  uuid: string; 
  firstName: string;
  lastName: string;
  email: string;
  createdBy: string;
  updatedBy: string;
  createdOn: Date;
  updatedOn: Date;
  dataProviderOrganisation?:DataProviderOrganisationRequest //uuid et name
}

